receiver/prometheusreceiver/internal/transaction.go

```diff
@@ -179,6 +180,10 @@ func (t *transaction) AppendExemplar(ref storage.SeriesRef, l labels.Labels, e e
 	return 0, nil
 }
 
+func (t *transaction) AppendHistogram(ref storage.SeriesRef, l labels.Labels, ts int64, h *histogram.Histogram) (storage.SeriesRef, error) {
+	return 0, fmt.Errorf("Not Implemented")
```

Comment:
Not sure whether this should return an error or `nil`.  Open to suggestions.

Reply:
IIRC, a failure to append one series in a scrape fails the entire scrape.  Do we know if that is the case here?   Seems safest to return nil.  We could debug log if we wanted to be helpful

Before we supported exemplars, we would return 0, nil in AppendExemplar: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f21f89eeb154975bf23fc74fabf0163e7626da28/receiver/prometheusreceiver/internal/otlp_transaction.go#L103